Package.swift

```diff
@@ -21,12 +23,15 @@
 import PackageDescription
 
 let package = Package(
-    name: "Starscream",
-    dependencies: [
-        .Package(url: "https://github.com/daltoniam/zlib-spm.git",
-    majorVersion: 1, minor: 1),
-        .Package(url: "https://github.com/daltoniam/common-crypto-spm",
-                 majorVersion: 1, minor: 1),
+        name: "Starscream",
+        products: [
+                .library(name: "Starscream", targets: ["Starscream"])
+        ],
+        dependencies: [
+                .package(url: "https://github.com/daltoniam/zlib-spm.git", from: "1.1.0"),
```

Comment:
I'm being picky here, but for consistency let's use same format without **.git**

Reply:
This is some merge issue, I will update my branch and discards this changes. 
I had no intention to change SPM config

Reply:
Looks like it's some GH issue, because these files are equal:

https://github.com/daltoniam/Starscream/blob/master/Package.swift
https://github.com/turbulem/Starscream/blob/master/Package.swift

I will make another branch and refresh this PR in order to make it clear.

Reply:
OK, I can adjust it later myself.